browse/services/listing/__init__.py

```diff
@@ -17,28 +17,31 @@
 """
 
 from typing import Optional
+from datetime import datetime
 
 from browse.domain.listing import NewResponse, ListingResponse
 
+
 class ListingService:
     """Class for arXiv document listings."""
 
     @classmethod
-    def version(self) -> str:
+    def version(cls) -> str:
```

Comment:
I think the python convention is to do this:

`__version__ = '0.2'`

Reply:
I switched it from self to cls because the linter was complaining. This pattern is from ABC and allows subclasses to return superclass independent version values.